docs/extensibility/form-section.md

```diff
@@ -354,7 +354,7 @@ MultiCheckbox widgets render checkboxes that are saved into one path as an array
   - **name** - an optional name for a checkbox instead of the default capitalized last part of the path. This can be a key from the **translation** section.
   - **description** - a string displayed in a tooltip when you hover over a question mark icon, next to the checkbox's label.
 - **required** - a boolean which specifies if a field is required. The default value is taken from CustomResourceDefintion (CRD); if it doesn't exist in the CRD, then it defaults to `false`.
-- **inputInfo** - a string below the input field that shows how to write in the input.
+- **inputInfo** - a string below the input field that shows how to write in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.
```

Comment:
```suggestion
- **inputInfo** - a string below the input field that shows how to write in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.